src/pages/index.tsx

```diff
@@ -1,41 +1,371 @@
+import CodeBlock from '@theme/CodeBlock';
+import Webcam from '@uppy/webcam';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+import IconReact from '../../static/img/react.svg';
+import IconVue from '../../static/img/vue.svg';
+import IconSvelte from '../../static/img/svelte.svg';
+import IconAngular from '../../static/img/angular.svg';
+import IconUpload from '../../static/img/upload.svg';
+import IconChat from '../../static/img/chat.svg';
+import IconLanguage from '../../static/img/language.svg';
+import IconSparkles from '../../static/img/sparkles.svg';
+import IconFolder from '../../static/img/folder.svg';
+import IconWrench from '../../static/img/wrench.svg';
+import IconUppy from '../../static/img/uppy.svg';
+
 import styles from './index.module.css';
-import HomepageFeatures from '@site/src/components/HomepageFeatures';
+import '@uppy/core/dist/style.min.css';
+import '@uppy/dashboard/dist/style.min.css';
 
-function HomepageHeader() {
-	const { siteConfig } = useDocusaurusContext();
-	return (
-		<header className={clsx('hero hero--primary', styles.heroBanner)}>
-			<div className="container">
-				<h1 className="hero__title">{siteConfig.title}</h1>
-				<p className="hero__subtitle">{siteConfig.tagline}</p>
-				<div className={styles.buttons}>
-					<Link
-						className="button button--secondary button--lg"
-						to="/docs/intro"
-					>
-						Docusaurus Tutorial - 5min ⏱️
-					</Link>
-				</div>
-			</div>
-		</header>
-	);
+const dashboardCode = `
+import Uppy  from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+import RemoteSources from '@uppy/google-drive'
+import ImageEditor from '@uppy/image-editor'
+import Webcam from '@uppy/webcam'
+import Tus from '@uppy/tus'
+
+const uppy = new Uppy()
+  .use(Dashboard, { target: '.DashboardContainer', inline: true })
+  .use(RemoteSources, { companionUrl: "http://companion.uppy.io" })
+  .use(Webcam, { target: Dashboard })
+  .use(ImageEditor, { target: Dashboard })
+  .use(Tus, { endpoint: 'https://tusd.tusdemo.net/files/' })
+`;
+
+const reactCode = `
+import React, { useEffect } from 'react'
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+import { Dashboard } from '@uppy/react'
+
+const uppy = new Uppy().use(Webcam)
+
+function Component () {
+  return <Dashboard uppy={uppy} plugins={['Webcam']} />
+} 
+`;
+
+const vueCode = `
+<template>
+  <div id="app">
+    <dashboard :uppy="uppy" :plugins="['Webcam']" :props="{theme: 'light'}" />
+  </div>
+</template>
+
+<script>
+import { Dashboard } from '@uppy/vue'
+
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+
+import Uppy from '@uppy/core'
+import Webcam from '@uppy/webcam'
+
+export default {
+  name: 'App',
+  components: {
+    Dashboard
+  },
+  computed: {
+    uppy: () => new Uppy().use(Webcam)
+  },
+  beforeDestroy () {
+    this.uppy.close({ reason: 'unmount' })
+  }
 }
+</script>
+`;
+
+const angularCode = `
+import { NgModule } from '@angular/core'
```

Comment:
```suggestion
const angularCode = `import { NgModule } from '@angular/core'
```

Would make the code start from line 1, currently it starts from line 2:

![Screenshot 14-03-2023 at 15 12 37@2x](https://user-images.githubusercontent.com/375537/225028489-6eca198f-c06a-441c-9feb-278d712001da.png)